mne/cov.py

```diff
@@ -386,66 +389,90 @@ def compute_raw_covariance(raw, tmin=None, tmax=None, tstep=0.2,
     picks : array-like of int | None (default None)
         Indices of channels to include (if None, all channels
         except bad channels are used).
+    keep_sample_mean : bool (default False)
```

Comment:
Actually I guess it's not from the `sample` dataset...

@agramfort @dengemann do you have any idea where the "erm cov" in our `mne/io/tests/data/test_erm-cov.fif` comes from? It doesn't match either from the `sample` dataset (according to a dumb test):
```
0fb0b2afb60ea4a6626b09ae142020bc MNE-sample-data/MEG/sample/ernoise-cov.fif
91cd4feea5bc8b18177351407d11787b MNE-sample-data/MEG/sample/sample_audvis-cov.fif
61a2d8033a3cabbed770adec6b2c5dde mne-python/mne/io/tests/data/test_erm-cov.fif
f8338b1cba1a8bf3a571ace3f72d97ff mne-python/mne/io/tests/data/test-cov.fif
```
Maybe we just computed it an incorrect way in the beginning and inherited it for `compute_covariance_raw`? So I guess changing the behavior to `keep_sample_mean=True` could be considered a bugfix?